platform/engineering/collab-cycle/architecture-intent-template.md

```diff
@@ -0,0 +1,56 @@
+# Architecture Intent Template
+
+__*** DRAFT *** DRAFT *** DRAFT ***__
+
+The Architecture Intent meeting helps your team build a solution that meets Veteran-facing Service Platform (VSP) engineering standards and lowers the potential for launch-blocking issues later in the development cycle.
+
+## What is the purpose of Architecture Intent meeting?
+
```

Comment:
I strongly agree with Bill here. That was a problem with Design Intent that we've tried to work on. Designers initially felt like they were going to be grilled, feared the process, and then either over prepared or waited to come until the work was farther along. We worked on our docs to let them know that it is a low stakes conversation rather than a rigorous critique. That's also part of the preamble that the host of the meeting reads out at the start.

Reply:
Would it ease the tension if these were more like 1:1 meetings rather than a team meeting with a panel?

Reply:
Agreed.  Added a TODO, will rework this in the next draft.

```
TODO: emphasize that this is a low stakes, collaborative conversation to help connect eng teams with the information and resources they need to build great things
```